src/main/java/io/logz/log4j2/LogzioAppender.java

```diff
@@ -348,7 +423,7 @@ private JsonObject formatMessageAsJson(LogEvent loggingEvent) {
 
     private static String getValueFromSystemEnvironmentIfNeeded(String value) {
         if (value == null)
-            return value;
+            return null;
```

Comment:
Why this change?

Reply:
@asafm I think it's clearer what is the return value

Reply:
right